doc/dune-files.rst

```diff
@@ -12,7 +12,7 @@ contents of all configuration files read by Dune and looks like:
 
 .. code:: scheme
 
-   (lang dune 2.8)
+   (lang dune 2.9)
```

Comment:
This is welcome but I think updating the docs to use the latest dune version should go in a separate PR.

Reply:
It's necessary for the tests to pass.

Reply:
I need this in another feature I'm working, so I did a separate PR: https://github.com/ocaml/dune/pull/4174